modules/architecture-machine-roles.adoc

```diff
@@ -42,18 +42,23 @@ Services that fall under the Kubernetes category on the master include the Kuber
 |Kubernetes API server
 |The Kubernetes API server validates and configures the data for pods, services,
 and replication controllers. It also provides a focal point for the shared state of the cluster.
+
 |etcd
 |etcd stores the persistent master state while other components watch etcd for
 changes to bring themselves into the specified state.
 //etcd can be optionally configured for high availability, typically deployed with 2n+1 peer services.
+
 |Kubernetes controller manager
 |The Kubernetes controller manager watches etcd for changes to objects such as
 replication, namespace, and service account controller objects, and then uses the
 API to enforce the specified state. Several such processes create a cluster with
 one active leader at a time.
+
+|Kubernetes scheduler
+|The Kubernetes scheduler watches for newly created pods and selects the best node for the pod to run on.
```

Comment:
consider "and selects the best node to run the pod." to de-clutter some prepositions.